### ISSUE TITLE
contributing: mention macOS requirements

### ISSUE THREAD

**Issue by myitcv** [play-with-go/play-with-go#163]
Take these from the macOS CI setup: https://github.com/play-with-go/play-with-go/blob/main/_scripts/macCISetup.sh

**Comment by myitcv** [comment]
We have moved #64 to "next" - for the same reason, moving this issue.

**Comment by maudel** [comment]
Can I take this as my first contribution?

**Comment by myitcv** [comment]
Thanks @maudel - before we explicitly list the macOS requirements, I want to remove any requirements that requires extra `brew` installs. So I'll put this on hold for now, then we can revisit.

Can I check: are you running macOS yourself?

**Comment by maudel** [comment]
Yes I am using macOs .... if that is the case can you recommend an issue that is good to start?

**Comment by myitcv** [comment]
> if that is the case can you recommend an issue that is good to start?

Of course. I've raised https://github.com/play-with-go/play-with-go/issues/305 as an action item in that regard.
